fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepositoryWrapper.java

```diff
@@ -240,6 +240,10 @@ public Loan findNonClosedLoanByAccountNumber(@Param("accountNumber") String acco
         return this.repository.findNonClosedLoanByAccountNumber(accountNumber);
     }
 
+    public boolean validateIfExistLoanWithExternalId(@Param("externalId") String externalId) {
```

Comment:
Same, method naming is a tiny bit confusing.
Bonus: why do we have the `@Param` annotation here? It just doesn't make any sense to me. Could you explain?

Reply:
Done!